torch/nn/parallel/distributed.py

```diff
@@ -216,6 +216,15 @@ class DistributedDataParallel(Module):
         unexpected behaviors can happen, since some parameters' gradient
         reduction functions might not get called.
 
+    .. warning::
+        Please noted that gradients are views of buffer tensors maintained by
+        DistributedDataParallel reducer since PyTorch 1.7. This can help save
+        one extra copy of grad tensors and reduce peak memory usage when using
+        DistributedDataParallel. Due to this change, detach_() cannot be called
+        on these gradients. If you hit error related to detach_() these gradients,
+        please refer to the Optimizer.zero_grad() function in
+        torch/optim/optimizer.py as the solution.
```

Comment:
can we convert this to a link? e.g.

```
please refer to the ``zero_grad()`` function in :class:`torch.optim.Optimizer` as a solution. 
```

Reply:
I am not 100% sure the above would work. Please confirm that by building the docs.